docs/source/cli/model-training-inference/configuration-run.rst

```diff
@@ -474,11 +474,21 @@ Link Prediction Task
     - Yaml: ``lp_decoder_type: dot_product``
     - Argument: ``--lp-decoder-type dot_product``
     - Default value: ``dot_product``
-- **gamma**: Gamma for ``DistMult``. The margin value in the score function.
+- **gamma**: Set the value of the hyperparameter denoted by the symbol gamma. Gamma is used in the following cases: i/ focal loss for binary classification ii/ DistMult score function for link prediction and iii/ RotatE score function for link prediction.
 
     - Yaml: ``gamma: 10.0``
     - Argument: ``--gamma 10.0``
-    - Default value: ``12.0``
+    - Default value: None
+- **alpha**: Set the value of the hyperparameter denoted by the symbol alpha. Alpha is used in the following case: i/ focal loss for binary classification.
+
+    - Yaml: ``alpha: 10.0``
+    - Argument: ``--alpha 10.0``
+    - Default value: None
+- **class_loss_func**: Node/Edge classification loss function. Builtin loss functions include ``cross_entropy`` and ``focal``.
```

Comment:
Can we add some links and explanations about what the alpha and gamma params control for each function?

Reply:
It is better to have a page about how to tune GraphStorm models with different settings.
Let's create a new task for it.

Reply:
Added this https://github.com/awslabs/graphstorm/issues/1004